app/app.go

```diff
@@ -168,64 +173,41 @@ func Run(ctx context.Context, config Config) error {
 				continue
 			}
 
-			log.Debug(ctx, "Signed exit")
-			signedExits = append(signedExits, obolapi.ExitBlob{
+			log.Info(ctx, "Signed partial exit")
```

Comment:
can also include validator index or share index as z field in this log, might help in debugging

Reply:
Good call!

Reply:
I forgot I already embedded this info in the context itself, you can see that on line 86.

The end result:

```
12:34:51.331 INFO            Signed partial exit                      {"validator": "0x83b9d210ae701b5be4d1931eee604ce4b268dfc85b0945ca29be8c945c727b8a9678e83f1fb8904082fae505b7d78586", "share_idx": 3, "peer_name": "ravishing-state"}
```